v1/flo.json

```diff
@@ -14,6 +14,10 @@
       "description": "With the device powered off, hold Volume Down + Power until the bootloader appears.",
       "image": "phone_power_down",
       "button": true
+    },
```

Comment:
You are missing a confirm_model entry here probably

Reply:
yep, thanks